platforms/android/library/src/main/java/io/element/android/wysiwyg/EditorEditText.kt

```diff
@@ -43,35 +43,66 @@ class EditorEditText : AppCompatEditText {
 
     private var inputConnection: InterceptInputConnection? = null
 
-    private lateinit var styleConfig: StyleConfig
+    var styleConfig: StyleConfig? = null
+        set(value) {
+            field = value
+
+            htmlConverter = value?.let { createHtmlConverter(it) }
```

Comment:
It's a public API so it should behave properly whatever value is supplied but at the moment setting to `null` doesn't seem to be supported.

Can we avoid making `StyleConfig` nullable? If it's absolutely necessary to make it nullable, perhaps we can handle a `null` value by applying the default style?